apps/web/stores/WalkthroughRootStore.ts

```diff
@@ -83,43 +79,19 @@ export class WalkthroughRootStore {
   }
 
   get currentQuestionAsVariable() {
-    const currentQuestion =
-      this.walkthroughData.questions[this.currentQuestionId];
+    const currentQuestion = this.walkthroughData.questions[this.currentItemId];
     if (!currentQuestion || !("variableToSet" in currentQuestion))
```

Comment:
It's not technically a part of this PR but I'm thinking things like this `"variableToSet"` should be added to a`constants.ts`.